### ISSUE TITLE
Cannot install Newman on Windows 10 x64

### ISSUE THREAD

**Issue by samuel-kollat** [postmanlabs/newman#264]
First my output:

```
C:\Users\CZ010129>npm install -g newman                                                                                 
> contextify@0.1.14 install C:\Users\CZ010129\AppData\Roaming\npm\node_modules\newman\node_modules\jsdom\node_modules\contextify                                                                                                                
> node-gyp rebuild                                                                                                                                                                                                                              /                                                                                                                      
 C:\Users\CZ010129\AppData\Roaming\npm\node_modules\newman\node_modules\jsdom\node_modules\contextify>if not defined npm_config_node_gyp (node "C:\Program Files (x86)\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node  rebuild )                                                                           
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.               contextify.cc                                                                                                         
C:\Users\CZ010129\AppData\Roaming\npm\node_modules\newman\node_modules\jsdom\node_modules\contextify\node_modules\nan\n an.h(263): error C2995: 'v8::Local<T> _NanEnsureLocal(v8::Local<T>)': function template has already been defined [C:\Us ers\CZ010129\AppData\Roaming\npm\node_modules\newman\node_modules\jsdom\node_modules\contextify\build\contextify.vcxpro j]                                                                                                                        
C:\Users\CZ010129\AppData\Roaming\npm\node_modules\newman\node_modules\jsdom\node_modules\contextify\node_modules\nan   \nan.h(256): note: see declaration of '_NanEnsureLocal'                                                               
C:\Users\CZ010129\AppData\Roaming\npm\node_modules\newman\node_modules\jsdom\node_modules\contextify\node_modules\nan\n an.h(661): error C3083: 'smalloc': the symbol to the left of a '::' must be a type [C:\Users\CZ010129\AppData\Roaming\n pm\node_modules\newman\node_modules\jsdom\node_modules\contextify\build\contextify.vcxproj]                             
C:\Users\CZ010129\AppData\Roaming\npm\node_modules\newman\node_modules\jsdom\node_modules\contextify\node_modules\nan\n an.h(661): error C2039: 'FreeCallback': is not a member of 'node' [C:\Users\CZ010129\AppData\Roaming\npm\node_modules\n ewman\node_modules\jsdom\node_modules\contextify\build\contextify.vcxproj]                                                
C:\Users\CZ010129\.node-gyp\4.0.0\include\node\node_object_wrap.h(8): note: see declaration of 'node'                 
C:\Users\CZ010129\AppData\Roaming\npm\node_modules\newman\node_modules\jsdom\node_modules\contextify\node_modules\nan\n an.h(661): error C2061: syntax error: identifier 'FreeCallback' [C:\Users\CZ010129\AppData\Roaming\npm\node_modules\new man\node_modules\jsdom\node_modules\contextify\build\contextify.vcxproj]                                                
C:\Users\CZ010129\AppData\Roaming\npm\node_modules\newman\node_modules\jsdom\node_modules\contextify\node_modules\nan\n an.h(665): error C2065: 'callback': undeclared identifier [C:\Users\CZ010129\AppData\Roaming\npm\node_modules\newman\no de_modules\jsdom\node_modules\contextify\build\contextify.vcxproj]                                                      
C:\Users\CZ010129\AppData\Roaming\npm\node_modules\newman\node_modules\jsdom\node_modules\contextify\node_modules\nan\n an.h(665): error C2065: 'hint': undeclared identifier [C:\Users\CZ010129\AppData\Roaming\npm\node_modules\newman\node_m odules\jsdom\node_modules\contextify\build\contextify.vcxproj]                                                          
C:\Users\CZ010129\AppData\Roaming\npm\node_modules\newman\node_modules\jsdom\node_modules\contextify\node_modules\nan\n an.h(672): error C2665: 'node::Buffer::New': none of the 4 overloads could convert all the argument types [C:\Users\CZ0 10129\AppData\Roaming\npm\node_modules\newman\node_modules\jsdom\node_modules\contextify\build\contextify.vcxproj]        C:\Users\CZ010129\.node-gyp\4.0.0\include\node\node_buffer.h(43): note: could be 'v8::MaybeLocal<v8::Object> node::Bu   ffer::New(v8::Isolate *,char *,size_t)'                                                                                 
C:\Users\CZ010129\.node-gyp\4.0.0\include\node\node_buffer.h(31): note: or       'v8::MaybeLocal<v8::Object> node::Bu   ffer::New(v8::Isolate *,v8::Local<v8::String>,node::encoding)'                                                          
C:\Users\CZ010129\AppData\Roaming\npm\node_modules\newman\node_modules\jsdom\node_modules\contextify\node_modules\nan   \nan.h(672): note: while trying to match the argument list '(v8::Isolate *, const char *, uint32_t)'                  
C:\Users\CZ010129\AppData\Roaming\npm\node_modules\newman\node_modules\jsdom\node_modules\contextify\node_modules\nan\n an.h(676): error C2440: 'return': cannot convert from 'v8::MaybeLocal<v8::Object>' to 'v8::Local<v8::Object>' [C:\Users \CZ010129\AppData\Roaming\npm\node_modules\newman\node_modules\jsdom\node_modules\contextify\build\contextify.vcxproj]    
C:\Users\CZ010129\AppData\Roaming\npm\node_modules\newman\node_modules\jsdom\node_modules\contextify\node_modules\nan   \nan.h(676): note: No constructor could take the source type, or constructor overload resolution was ambiguous       
 C:\Users\CZ010129\AppData\Roaming\npm\node_modules\newman\node_modules\jsdom\node_modules\contextify\node_modules\nan\n an.h(683): error C2039: 'Use': is not a member of 'node::Buffer' [C:\Users\CZ010129\AppData\Roaming\npm\node_modules\ne wman\node_modules\jsdom\node_modules\contextify\build\contextify.vcxproj]                                                 
C:\Users\CZ010129\.node-gyp\4.0.0\include\node\node_buffer.h(8): note: see declaration of 'node::Buffer'              
C:\Users\CZ010129\AppData\Roaming\npm\node_modules\newman\node_modules\jsdom\node_modules\contextify\node_modules\nan\n an.h(683): error C3861: 'Use': identifier not found [C:\Users\CZ010129\AppData\Roaming\npm\node_modules\newman\node_mod ules\jsdom\node_modules\contextify\build\contextify.vcxproj]                                                            
gyp ERR! build error                                                                                                    
gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe` failed with exit code: 1                    
gyp ERR! stack     at ChildProcess.onExit (C:\Program Files (x86)\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:270:23)                                                                                                            
gyp ERR! stack     at emitTwo (events.js:87:13)                                                                         
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)                                                               
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)                            
gyp ERR! System Windows_NT 10.0.10240                                                                                   
gyp ERR! command "C:\\Program Files (x86)\\nodejs\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"                                                                           
gyp ERR! cwd C:\Users\CZ010129\AppData\Roaming\npm\node_modules\newman\node_modules\jsdom\node_modules\contextify       gyp ERR! node -v v4.0.0                                                                                                 
gyp ERR! node-gyp -v v3.0.1                                                                                            
gyp ERR! not ok                                                                                                         
npm ERR! Windows_NT 10.0.10240                                                                                          
npm ERR! argv "C:\\Program Files (x86)\\nodejs\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "newman"                                                                                         
npm ERR! node v4.0.0                                                                                                    
npm ERR! npm  v2.14.2                                                                                                  
npm ERR! code ELIFECYCLE                                                                                                                                                                                                                        npm ERR! contextify@0.1.14 install: `node-gyp rebuild`                                                                  
npm ERR! Exit status 1                                                                                                  
npm ERR!                                                                                                                
npm ERR! Failed at the contextify@0.1.14 install script 'node-gyp rebuild'.                                             
npm ERR! This is most likely a problem with the contextify package,                                                     
npm ERR! not with npm itself.                                                                                           
npm ERR! Tell the author that this fails on your system:                                                                
npm ERR!     node-gyp rebuild                                                                                           
npm ERR! You can get their info via:                                                                                    
npm ERR!     npm owner ls contextify                                                                                    
npm ERR! There is likely additional logging output above.                                                                                                                                                                                       
npm ERR! Please include the following file with any support request:                                                    
npm ERR!     C:\Users\CZ010129\npm-debug.log  
```

Node version: v4.0.0
NPM version: 2.14.2
OS: Windows 10 Enterprise
Visual Studio version: VS Professional 2015 (14.023107.0)

I even tried --msvs_version=2015, but it didn't help.

**Comment by czardoz** [comment]
Node v4 is not supported in Newman yet. We're working on it in this branch:

https://github.com/postmanlabs/newman/tree/feature/node-v4-support

Pull requests are welcome :)

**Comment by DraXus** [comment]
@czardoz it would be great to have a warning note in the README about that. It would save us a lot of time figuring out why is not working :)

**Comment by czardoz** [comment]
@DraXus, we've released a beta for Node v4.0+, see the updated [readme](https://github.com/postmanlabs/newman/) for details.
